src/UI/EditorDiffMarkers.rei

```diff
@@ -0,0 +1,27 @@
+open Oni_Core;
+
+[@deriving show]
+type t = array(marker)
+
+and marker =
+  | Modified
+  | Added
+  | DeletedBefore
+  | DeletedAfter
+  | Unmodified;
```

Comment:
These are only exposed for testing purposes. I wish there was a better way of testing that didn't require breaking encapsulation.

Reply:
Ah, right, I wish I had this too - sometimes I want to expose types just for tests, but not as part of the 'public' API.